package.json

```diff
@@ -59,29 +59,28 @@
     "@oclif/command": "^1.0",
     "@oclif/config": "^1.12.11",
     "@oclif/plugin-help": "^2.0",
-    "@stoplight/json": "^2.1.0",
-    "@stoplight/json-ref-resolver": "^1.5.0",
-    "@stoplight/types": "^7.0.1",
-    "@stoplight/yaml": "^2.3.0",
+    "@stoplight/json": "2.x.x",
```

Comment:
Ditto, let's use ^ syntax as Will described in Graphite PR.

Reply:
That's a huge change that we should agree upon as a team, otherwise we'll end up being more inconsistent than we are now. Why ^ versus x? Can you link me to PR?

Reply:
https://github.com/stoplightio/graphite/pull/194#discussion_r294107274

Reply:
Done